fortune_generator/js/fortune.js

```diff
@@ -227,7 +293,8 @@ function Appear() {
 
   if (!fortune_generated) {
     // transform ip to four numbers
-    const num = ip.split(".").map((num) => parseInt(num));
+    // const num = ip.split(".").map((num) => parseInt(num));
+    const num = [36, 236, 233, 221]; // NOTE: for dev
```

Comment:
It looks like the line for splitting and parsing the IP address is commented out, and a hardcoded value is used instead. Please remove this line.